packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -35,9 +48,11 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     // Use CryptoWatch's most granular option, one minute.
     this.ohlcPeriod = 60;
 
-    // Helper functions from web3.
-    this.toBN = this.web3.utils.toBN;
-    this.toWei = this.web3.utils.toWei;
+    this.convertDecimals = number => {
+      // Converts price result to wei
+      // returns price conversion to correct decimals as a big number
+      return parseFixed(number.toString(), decimals);
```

Comment:
Does this return the same type of `BN` you are expecting? If not might be worth casting to the `toBN` you use

Reply:
i think this actually fixes the ci issue!